frontend/src/main/components/Commons/FarmStats.js

```diff
@@ -1,25 +1,31 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
 import React from "react";
 import { Card } from "react-bootstrap";
+import Cash from "./../../../assets/Cash.png";
+import Health from "./../../../assets/Health.png";
 
 const FarmStats = ({userCommons}) => {
    
     return (
         <Card>
-        <Card.Header as="h5">Your Farm Stats</Card.Header>
+        <Card.Header as="h5"><strong>Your Farm Stats</strong></Card.Header>
```

Comment:
You could also reuse the previously suggested CSS class here.

Reply:
Tyler: at first I was agreeing with you, and even went so far as to look up "what is the bootstrap css class that would apply here".

To my surprise, I found that the way Isaiah wrote the code here actually conforms to what Bootstrap recommends in their own CSS documentation linked here: https://getbootstrap.com/docs/3.4/css/#bold

So, I'm thinking: maybe it's ok to leave it?

I do see your point though, that if we made a CSS class called `.commonsHeaders`, that it would make it easier to style these headers in a single place.  I'll leave it up to the team to decide which way to go.

Reply:
Hi @pconrad, that makes sense!

In my review, I assumed that future changes to the styling of all headers was likely (say, increasing font size or adding some other text decoration), so my intuition was to create a CSS class that could be further augmented in a single location.

However, this was just my own intuition; I defer to you and the Bootstrap standard.

I would be happy to approve this as-is when my review is re-requested. Thank you for weighing in!

Reply:
Hi @pconrad !

The decision was pretty much left up to me after getting feedback from the team. I decided to implement some classes labeled `.icon` and `h5` in index.css.

`.icon` is a class to specify image features. There is only `width` and `height`, which are specified to 'auto' . 

`h5` manipulates the h5 tag by simply using the `font-weight` parameter. I understand that doing this affects ANY text that is specified with an h5 tag, but I think all headers should be bolded anyways. Also, since we are using `Card.Header a="h5`, it makes more sense to abstract the bolding feature into the h5 class itself. 

Thank you for all the feedback, though! It has helped me brush up some knowledge on CSS classes.

Reply:
@tekaireb ^^ This message also applies to what you suggested. Thanks Tyler!